corehq/apps/api/tests.py

```diff
@@ -420,7 +420,7 @@ def test_update(self):
         response = self.client.put(self.single_endpoint(backend_id),
                                    simplejson.dumps(user_json),
                                    content_type='application/json')
-        self.assertEqual(response.status_code, 202, response.content)
+        self.assertEqual(response.status_code, 200, response.content)
```

Comment:
Changed in Tastypie v0.9.16: https://github.com/django-tastypie/django-tastypie/commit/abc0bef